src/index.ts

```diff
@@ -121,6 +121,7 @@ export * from "./decorator/controllers";
 export * from "./decorator/decorators";
 export * from "./decorator/methods";
 export * from "./decorator/params";
+export * from "./error/http/HttpError";
```

Comment:
You are right! I've done it in 6d16e78.
Now you can import them also this way:
`import { HttpError } from "routing-controllers/error";`